docs/ovs_communication.adoc

```diff
@@ -120,14 +135,16 @@ b. start vswitchd: ovs-vswitchd -v --pidfile --detach \
 
 c. use cmd to init: ovs-vsctl --no-wait init
 ------------------------------------------------------------
-
+option 2 is start the openvswitch service directly which would do the same to option 1, btw option 2 is the neutron way
```

Comment:
We will likely go with option 2 then, can you please share the corresponding code in neutron?

Reply:
please refer to 
https://github.com/openstack/neutron/blob/c2d18cda6f3716965f0843da213985b4b7c6bb41/devstack/lib/ovs
line 153